packages/backend-storage/src/construct.ts

```diff
@@ -42,7 +44,10 @@ export type AmplifyStorageProps = {
    * @default false
    */
   versioned?: boolean;
-  outputStorageStrategy?: BackendOutputStorageStrategy<StorageOutput>;
+  outputStorageStrategy?: BackendOutputStorageStrategy<{
+    version: '1';
+    payload: Record<string, string>;
```

Comment:
I think @sobolk had a similar comment elsewhere, but we should keep this strongly typed, even if that means updating the type